com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -7,7 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [7.5.2] - 2020-09-14
 
 ### Fixed
+- Fixed not using the local skybox on the camera game object when Lighting>environment skybox was set to null.
+- Fixed an issue with a render texture failing assertion when chosing an invalid format. [case 1271034](https://issuetracker.unity3d.com/issues/the-error-occurs-when-a-render-texture-which-has-a-certain-color-format-is-applied-to-the-cameras-output-target)
```

Comment:
Seems like these two were moved unintentionally?

Reply:
Sorry for not mentioning this in the PR description.

The PR's that added those lines came in after 7.5.1 was released. So they were incorrectly placed and will be in 7.5.2.
Decided to fix those in this PR as well.